src/paste-markdown-html.ts

```diff
@@ -95,6 +106,29 @@ function convertToMarkdown(plaintext: string, walker: TreeWalker): string {
   return index === NODE_LIMIT ? plaintext : markdown
 }
 
+function getNodeMarkdownBuilder(node: Node): (() => string) | void {
```

Comment:
This function returns a function to make it lazy. This allows us to see if we _support_ converting a specific `Node` without actually spending the energy to convert it.